README.md

```diff
@@ -26,6 +26,7 @@ Example PR here (click "View details"): <https://github.com/hchiam/learning-ligh
     and in a separate CLI tab:
 
     ```bash
+    npm build
```

Comment:
Thanks for catching this! I tested it out locally just now and it definitely now needs the build script to run first. We'll also need the `run` keyword:

```suggestion
    npm run build
```

Reply:
nice! thank you 🚀